airgun/entities/host_new.py

```diff
@@ -178,6 +178,142 @@ def get_tracer(self, entity_name):
         self.browser.plugin.ensure_page_safe()
         return view.traces.read()
 
+    def get_os_info(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        return view.details.operating_system.read()
+
+    def get_provisioning_info(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        return view.details.provisioning.read()
+
+    def get_bios_info(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        return view.details.bios.read()
+
+    def get_registration_details(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        return view.details.registration_details.read()
+
+    def get_hw_properties(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        return view.details.hw_properties.read()
+
+    def get_provisioning_templates(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        d = view.details.read()
+        return d['provisioning_templates']['templates_table']
+
+    def get_networking_interfaces(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        return view.details.networking_interfaces.networking_interfaces_accordion.items()
+
+    def get_networking_interfaces_details(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
+        self.browser.plugin.ensure_page_safe()
+        net_devices = [
+            i.split()[0]
+            for i in view.details.networking_interfaces.networking_interfaces_accordion.items()
+        ]
+
+        for dev in net_devices[1:]:
+            view.details.networking_interfaces.networking_interfaces_accordion.toggle(dev)
+
+        def val_gen(item_name):
+            """Generator for values in the networking interfaces accordion"""
```

Comment:
here the name sounds a bit misleading, does it really generate values?

Reply:
for the dictionary yes

Reply:
but not in the way that they are not in the UI. I needed to generate them one by one for each interface that is saved in the dictionary

Reply:
I can change the comment